### PR TITLE
[win] Remove compiler flags workarounds

### PR DESCRIPTION
**Pull request by bellenot**
Those workarounds are not needed anymore with the latest VS 2019 and VS 2022

### CODE REVIEW

**Comment by bellenot** [comment]
@phsft-bot build just on windows10/cxx14

**Comment by bellenot** [comment]
@phsft-bot build just on windows10/cxx14

File: tree/dataframe/test/dataframe_interface.cxx [hunk 1]
```diff
@@ -641,7 +641,7 @@ TEST(RDataFrameInterface, Describe)
 
    // dataframe with various data types
    RDataFrame df2(tree);
-   auto df3 = df2.Define("myVec", "ROOT::RVec<float>({1, 2, 3})")
+   auto df3 = df2.Define("myVec", "::ROOT::RVec<float>({1, 2, 3})")
```
[the remark this answers is not in the document]
Comment (eguiraud) [reply tree/dataframe/test/dataframe_interface.cxx:644]: Or let me formulate this better: if this doesn't work, there should be many more places where we need to make a similar change, including tutorials. But I think it's a problem if we have to teach all users to prepend `::` to their qualified names, it shouldn't be the case. IMHO it's something to fix in cling+windows

[the remark this answers is not in the document]
Comment (bellenot) [reply tree/dataframe/test/dataframe_interface.cxx:644]: > uhm, this should work as it was written..?

Don't worry, I'm just trying (debugging right now)

**Review by Axel-Naumann — APPROVED** [review]

**Pull request merged**